frontend/packages/dev-console/src/components/import/ImportForm.tsx

```diff
@@ -138,6 +138,7 @@ const ImportForm: React.FC<ImportFormProps & StateProps> = ({
     },
     import: {
       loaded: false,
+      kNativeFuncLoaded: false,
```

Comment:
NIT: lets call it `knativeFuncLoaded`. `Knative` is a single word. 
```suggestion
      knativeFuncLoaded: false,
```